packages/design-editor/src/shell/shell.css

```
@@ -0,0 +1,83 @@
+    sans-serif;
+  --font-mono:
+    ui-monospace, SFMono-Regular, 'SF Mono', Menlo, Consolas, 'Liberation Mono', monospace;
+}
+
+/*
+ * The shell follows the DEVELOPER's OS preference, not the theme being edited.
+ * Those are different things: previewing a dark theme must not flip the chrome, or
+ * the two are no longer distinguishable on screen.
+ */
+@media (prefers-color-scheme: dark) {
+  @theme {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@theme`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)